src/operations/run_command.ts

```diff
@@ -44,6 +48,10 @@ export class RunAdminCommandOperation<T = Document> extends AbstractOperation<T>
     this.ns = new MongoDBNamespace('admin', '$cmd');
   }
 
+  override get commandName() {
+    return 'runAdminCommand' as const;
```

Comment:
```suggestion
    return 'runCommand' as const;
```

These are the two "pseudo" command names. runAdminCommand is not something that is a MongoDB at large concept, just a differentiation in our driver that isn't relevant to the server selection logging. I think we should stick with the shared concept of "runCommand" here.